master/pkg/model/experiment.go

```diff
@@ -522,6 +522,7 @@ type Run struct {
 	LogRetentionDays      *int16         `db:"log_retention_days"`
 	Metadata              map[string]any `db:"metadata" bun:"metadata,scanonly"`
 	LocalID               int            `db:"local_id"`
+	LogPolicyMatched      *string        `db:"log_policy_matched"`
```

Comment:
Why do we want to add `LogPolicyMatched` to experiment model?

Reply:
I thought `/api/v1/runs` query uses this model. But checking the code again, I see that it actually [uses the go struct ](https://github.com/determined-ai/determined/blob/63d728cf0519f89fee2993e9530e17321a946819/master/internal/api_runs.go#L89-L93)generated from `message Flatrun` in the proto file. Thank you, great catch!